### PR TITLE
Draft: Add support for remote playbooks in prepare

### PR DESCRIPTION
**Pull request by happz**
Besides the local ones, `ansible` plugin would also allow remote
playbooks.

Playbooks are fetched and saved, for the record, then given to Ansible
as any other local playbook.

### CODE REVIEW

**Comment by happz** [comment]
FYI, @psss, based on our little brainstorming.

File: tmt/steps/prepare/ansible.py [hunk 1]
```diff
@@ -74,4 +88,29 @@ def go(self, guest):
...
+
+                except requests.RequestException as exc:
+                    raise PrepareError(
+                        f"failed to fetch remote playbook '{playbook}'", original=exc)
+
+                with tempfile.NamedTemporaryFile(mode='w+b', prefix='playbook-', suffix='.yml', dir=None, delete=False) as f:
+                    f.write(response.content)
+                    f.flush()
+
+                    playbook_path = f.name
```
[the remark this answers is not in the document]
Comment (happz) [reply tmt/steps/prepare/ansible.py:112]: Indeed, it does work, thanks.

**Comment by psss** [comment]
Although this would not support playbooks depending/importing other playbooks, for the basic use case could still be useful and the implementation is quite straightforward. I'd propose to finish it and include in `1.17`.

**Comment by happz** [comment]
Closing in favor of https://github.com/teemtee/tmt/pull/1489 which is not based in a fork (too much work to rebase this PR...)

**Pull request closed without merging**